mavros/src/plugins/local_position.cpp

```diff
@@ -91,16 +95,15 @@ class LocalPositionPlugin : public MavRosPlugin {
 				pos_ned.x, pos_ned.y, pos_ned.z,
 				pos_ned.vx, pos_ned.vy, pos_ned.vz);
 
-		/* TODO: check convertion to ENU
-		 * I think XZY is not body-fixed, but orientation does.
-		 * Perhaps this adds additional errorprone to us.
-		 * Need more tests. Issue #49.
-		 *
-		 * orientation in ENU, body-fixed
-		 */
+		geometry_msgs::QuaternionStamped q;
+		tf::quaternionTFToMsg(uas->get_attitude_orientation(), q.quaternion);
+		q.header.frame_id = child_frame_id;
+		q.header.stamp = uas->synchronise_stamp(pos_ned.time_boot_ms);
+		tf_listener.transformQuaternion(frame_id, q, q);
+
 		tf::Transform transform;
 		transform.setOrigin(tf::Vector3(pos_ned.y, pos_ned.x, -pos_ned.z));
```

Comment:
Shouldn't this be `transform.setOrigin(tf::Vector3(pos_ned.x, -pos_ned.y, -pos_ned.z));`